tests/assets/move-projects/balance/sources/Scripts.move

```diff
@@ -0,0 +1,9 @@
+script {
+    use substrate::balance;
+
+    fun get_preconfigured_balance(account: address, preconfigured_amount: u128) {
```

Comment:
`get_preconfigured_balance()` confuses me a little, since there is nothing to be returned - only to verify/check some current setup.

Reply:
Since scripts cannot return values, the name should have meant something like (in this context):

    // Now check that it works from within the MoveVM.
    let script = assets::read_script_from_project("balance", "get_preconfigured_balance");

_Call a script that does exactly this: "get_preconfigured_balance". Otherwise - it shall fail..._

But I dunno - do you have any suggestions here?

Reply:
`check_get_balance_API_with_a_preconfigured_balance` maybe?

Reply:
Yes, or maybe `verify_preconfigurd_balance`? You decide :blush:

Reply:
Uuu - I like that one more :fire:
Feels like a more imperative approach.

Reply:
Done